andela/Ah-frontend-xmen
# #162725216: Integrate sass into the application

#### What does this PR do?
* Sets up SASS processing on the client
#### Description of Task to be completed?
* Install and setup sass pre-processing on the application to enables the use of both .scss and .css files on the client
#### How should this be manually tested?
* fetch this branch by running `git fetch origin ch-integrate-sass-processing-162725216` in your terminal
* check out into the branch using `git checkout ch-integrate-sass-processing-162725216`
* run `yarn install` to install the dependencies
* run `yarn start` to start the development server
* navigate to localhost:8000
* view the sass effect on the different page elements
#### What are the relevant pivotal tracker stories?
[Starts #162725216](https://www.pivotaltracker.com/n/projects/2232225/stories/162725216https://www.pivotaltracker.com/n/projects/2232225/stories/162725216https://www.pivotaltracker.com/n/projects/2232225/stories/162725216)
